docs/devel/releasing.md

```diff
@@ -93,21 +93,21 @@ commits,) so choose the latest build on the release branch.  (Remember, that
 branch should be frozen.)
 
 Once you find some greens, you can find the git hash for a build by looking at
-the Full Console Output and searching for `githash=`. You should see a line:
+the Full Console Output and searching for `GIT_COMMIT_HASH=`. You should see a line:
```

Comment:
Where does this string get printed?  I don't see a change in this PR that changes a printout from "githash" to "GIT_COMMIT_HASH"